requirement-generator/requirement-sources-jsons.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * The module that re-exposes the generated jsons with strict static types.
+ */
+
+import { readFileSync } from 'fs';
+import { Course, RequirementsJson } from './types';
+
+/** All courses fetch from Cornell Class API, with only the fields we need. */
+export const filteredAllCourses: { readonly [semester: string]: readonly Course[] } = JSON.parse(
+  readFileSync('requirement-generator/filtered-all-courses.json').toString()
+);
+
+/** The requirement json that guides our algorithm to decide whether a class satisfies a requirement. */
+export const requirementJson: RequirementsJson = JSON.parse(
+  readFileSync('src/requirements/reqs.json').toString()
```

Comment:
Is readFileSync and toString() the best option compared to `require('src/requirements/reqs.json')`

Reply:
A little technical background: this `requirement-generator` folder is currently an isolated project from the `src/` folder. Using `require` will break typescript import. In the future, I plan to merge them together when things are stabilized.

Reply:
That's fair thank you